en/README.md

```diff
@@ -0,0 +1,1138 @@
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchTriggers(method)
+
+This procedure request to get triggers to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched triggers using a putTriggers procedure. At that time, need to pass fetchId and hostIds of fetchTriggers procedure params to putTriggers procedure.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostIds|String255 array|O|-|Get triggers of to designate a hosts. If does not designate hostIds, all trigger are targeted.|
+|fetchId|String255    |M|-|Use in putTriggers procedures. It need to identify the putTriggers corresponding to the fetchTriggers.|
+
+
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "fetchTriggers",
+  "params": {
+    "hostIds": [
+      "1",
+      "2"
+    ],
+    "fetchId": "1"
+  },
+  "id": 1
+}
+```
+
+***Result(result)***
+
+Return result as result object value whether request accept to Hatohol server. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchEvents(method)
+
+This procedure request to get designate number of events that sorted by ascending or descending order to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched event using a putEvents procedure. At that time, need to pass fetchId of fetchEvents procedure params to putEvents procedure.
+
+Can designate "count" value till 1000. If want to request over the event number 1000, separate request to more than once.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|lastInfo |String32767|M|-|Basis of ivent information.|
+|count    |Number   |M|-|Event count of getting.|
+|direction|String255|M|-|Select "ASC"(newer event than designate ID) or $B!I(BDESC$B!I(B(older event that designate ID).|
+|fetchId  |String255|M|-|Use in putEvents procedures. It need to identify the putEvents corresponding to the fetchEvents.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "direction": "ASC",
+    "count": 1000,
+    "lastInfo": "10",
+    "fetchId": "1"
+  },
+  "method": "fetchEvents",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Return result as result object value whether request accept to Hatohol server. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### updateMonitoringServerInfo(notification)
+
+ This procedure is sent from Hatohol server when updated monitoring server information. At that time, should update each monitoring server information to restart HAP.
+
+***params***
+
+params content is same as the result object of [getMonitoringServerInfo](#user-content-getmonitoringserverinfomethod).
```

Comment:
> params content is same as the result object of [getMonitoringServerInfo](#user-content-getmonitoringserverinfomethod).

Good sentence. ;)

Perhaps, this sentence is also written as:

Params content is identical to the result object of [getMonitoringServerInfo](#user-content-getmonitoringserverinfomethod).